core/templates/components/ck-editor-helpers/ck-editor-copy-content.service.spec.ts

```diff
@@ -77,7 +77,7 @@ describe('Ck editor copy content service', () => {
     service.toggleCopyMode();
     expect(service.copyModeActive).toBe(true);
 
-    const pElement = generateContent('<p>Hello</p>');
+    const pElement = generateContent('<p>Hello<!--comment--></p>');
```

Comment:
Why is this needed?

Reply:
The changes in `ck-editor-copy-content.service.ts` result in the frontend tests not being fully covered and failing without testing that comments are removed: 
<img width="775" alt="Screen Shot 2022-12-19 at 3 23 37 PM" src="https://user-images.githubusercontent.com/85202273/208514781-e21539e6-052d-4818-bd12-98c5746d8378.png">

Reply:
Ah, okay